plane/plane-tests/tests/common/test_env.rs

```diff
@@ -126,8 +128,8 @@ impl TestEnvironment {
             ip: IpAddr::V4(Ipv4Addr::LOCALHOST),
             db_path: Some(self.scratch_dir.join("drone.db")),
             pool: pool.clone(),
-            auto_prune: false,
-            cleanup_min_age: Duration::zero(),
+            auto_prune: None,
+            cleanup_min_age: None,
```

Comment:
these are deprecated; while they are still around, the values inside the docker config will take precedence if specified.